free-courses-en.md

```diff
@@ -79,11 +79,13 @@
 
 ### Clojure
 
+* [BraveClojure](http://www.braveclojure.com)
```

Comment:
Is this a course or a book? If it's a book, could you please move this link to `free-programming-books.md` and directly link to the book?